core/api/tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "sourceMap": true,
     "strict": true,
     "target": "es2020",
-    "noImplicitAny": false,
+    "noImplicitAny": true,
```

Comment:
it's also the default when `strict: true` so could be removed

Reply:
Unfortunately it won't work due to the tests.